<!DOCTYPE html>
<!-- saved from url=(0059)http://www.fbs.leeds.ac.uk/staff/profile.php?tag=OConnell_M -->
<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="FBS staff profile for Dr Mary O&#39;Connell">
    <meta name="keywords" content="Mary O&#39;Connell">
    <meta name="author" content="Faculty of Biological Sciences, University of Leeds">
    <link href="./Dr Mary O&#39;Connell_files/css" rel="stylesheet" type="text/css">
    <link rel="icon" href="http://www.fbs.leeds.ac.uk/favicon.ico">

    <title>Dr Mary O'Connell</title>

    <!-- Bootstrap core CSS -->
    <link href="./Dr Mary O&#39;Connell_files/bootstrap.min.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <link href="./Dr Mary O&#39;Connell_files/bootstrap-override.css" rel="stylesheet">
      
    <!-- font awesome icons -->
    <link rel="stylesheet" href="./Dr Mary O&#39;Connell_files/font-awesome.min.css">

    <link href="./Dr Mary O&#39;Connell_files/profile.css" rel="stylesheet"><link href="./Dr Mary O&#39;Connell_files/fullwidth.css" rel="stylesheet">

    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!--<script src="//code.jquery.com/jquery-2.1.4.min.js"></script>-->
    <style type="text/css">img[src="../php/emailToGIF.php?ad=fbsmo"]
{display:none !important;}</style><script async="" src="http://www.googletagmanager.com/gtm.js?id=GTM-WT437X"></script><script async="" src="http://www.google-analytics.com/analytics.js"></script><script src="./Dr Mary O&#39;Connell_files/jquery.min.js.download"></script>

    <script src="./Dr Mary O&#39;Connell_files/bootstrap.min.js.download"></script>
    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
    <script src="./Dr Mary O&#39;Connell_files/ie10-viewport-bug-workaround.js.download"></script>

    <!-- any extra scripts for the CMS -->
    <script src="./Dr Mary O&#39;Connell_files/bootstrap-override.js.download"></script>

    
    <script>
    $(function() {
        // handle tab clicks
        $('ul.nav a').click(function (e) {
            var hash = $(this).attr('href');
            $("div .tabset-content > div").addClass("hide");
            $(hash).removeClass("hide");
        });
    });

    // handle the clicking of the areas with class .editable
    $(document).on("click", ".editable", function(e) {
        var url = $(this).attr("rel");
        if (url == '' || typeof(url) == "undefined") {
            url = $(this).data("link");
        }
        if (url != '' && typeof(url) != "undefined") {
            document.location = url;
        }

        e.preventDefault(); // the link may be on an object between <a></a> tags
    });

        </script>
  </head>

  <body>

    <!-- Google analytics -->    <script>
    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

    ga('create', 'UA-9440306-2', 'leeds.ac.uk');

            ga('set', 'dimension1', '9PG5eRnagBeEVirpF8h51Q==');
        
    ga('send', 'pageview');


	/**
	* Function that tracks a click on an outbound link in Analytics.
	* This function takes a valid URL string as an argument, and uses that URL string
	* as the event label. Setting the transport method to 'beacon' lets the hit be sent
	* using 'navigator.sendBeacon' in browser that support it.
	*/
	var trackOutboundLink = function(url) {
	   ga('send', 'event', 'outbound', 'click', url, {
		   'transport': 'beacon',
		   'hitCallback': function(){document.location = url;}
	   });
	}
    </script>
    <!-- Google Tag Manager --> 
	<noscript>&lt;iframe src="//www.googletagmanager.com/ns.html?id=GTM-WT437X" 
	height="0" width="0" style="display:none;visibility:hidden"&gt;&lt;/iframe&gt;</noscript> 
	<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start': 
	new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0], 
	j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src= 
	'//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f); 
	})(window,document,'script','dataLayer','GTM-WT437X');</script> 
	<!-- End Google Tag Manager -->

    <div class="container">
      <div class="header">
        <nav class="center-block">
          <div class="pull-left">
            <div class="small shortcuts"><a href="http://www.leeds.ac.uk/" rel="external">Leeds.ac.uk</a> | <a href="http://www.fbs.leeds.ac.uk/intranet" rel="external">Intranet</a> | <a href="https://leedsportal.leeds.ac.uk/" rel="external">Portal</a> | <a href="http://www.fbs.leeds.ac.uk/staff/contacts.htm" rel="external">Contact</a></div>
                      </div>
          <span class="pull-right">
            <a href="http://www.leeds.ac.uk/" rel="external">
              <img id="unilogo" src="./Dr Mary O&#39;Connell_files/uol-logo.svg" width="183" height="54" alt="University of Leeds">
            </a>
          </span>
        </nav>
      </div>


      <div class="jumbotron center-block">
        <div class="branding">
          <div class="ident">
            <h3 style="background-color: #1C5689">
              Faculty of Biological Sciences            </h3>
          </div>

          <nav id="fbs-navbar" class="navbar navbar-default" role="navigation">
            <div class="navbar-header">
              <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
              </button>
              <!--<a class="navbar-brand" href="#">Project name</a>-->
            </div>
                          <div id="navbar" class="navbar-collapse collapse">
              <ul class="nav navbar-nav">
                                <li><a href="http://www.fbs.leeds.ac.uk/">Home</a></li>
                <li><a href="http://www.fbs.leeds.ac.uk/about.htm">About</a></li>
                <li><a href="http://www.fbs.leeds.ac.uk/undergraduate/">Undergraduate</a></li>
                <li><a href="http://www.fbs.leeds.ac.uk/postgraduate/">Postgraduate</a></li>
                <li><a href="http://www.fbs.leeds.ac.uk/research/">Research and Innovation</a></li>
                <li><a href="http://www.fbs.leeds.ac.uk/outreach/">Outreach</a></li>
                <li><a href="http://www.fbs.leeds.ac.uk/alumni/">Alumni</a></li>
                <li><a href="http://www.fbs.leeds.ac.uk/staff/" class="active">People</a></li>
                              </ul>
            </div>
          </nav>
        </div><!-- branding -->


        <div class="breadcrumb hidden-xs"><a name="top"></a>
                  <p>
            <span class="small">You are here: <a href="http://www.fbs.leeds.ac.uk/" rel="external">Home</a> &gt;
            <a href="http://www.fbs.leeds.ac.uk/staff/" rel="external">Staff</a> &gt; Dr Mary O'Connell            </span>
          </p>
                </div>

        
      </div><!-- jumbotron -->
    </div><!-- container -->

    <div class="container">
      <div class="jumbotron center-block">
        <div class="row">
			<!-- sidebar -->
			<div class="col-sm-2">
				<div id="secondaryNav"><h2>Staff Pages</h2>
<ul class="navList">
  <li><a href="http://www.fbs.leeds.ac.uk/staff/index.htm">Staff Search page</a>  </li>
  <li><a href="http://www.fbs.leeds.ac.uk/staff/contacts.htm">Key Faculty Contacts</a>  </li>
  <li><a href="http://www.fbs.leeds.ac.uk/staff/alpha.htm">Alphabetical Staff list</a></li>
</ul>
</div>
			</div>
			<!-- page content -->
			<div id="page-content" class="col-sm-10"><div id="person_info" style="min-height:160px"><div class="staff_photo" style="background-image:url(/php/showStaffPhoto.php?un=fbsmo); background-size: contain;"></div><h3>Dr Mary O'Connell</h3><h4>BSc (Hons) PhD<br>University Academic Fellow in Biology<br>School of Biology<br></h4><p><strong>Background</strong>: <i>My PhD (2005) addressed the role of positive selection in molecular evolution and assessed rate heterogeneity in protein coding regions of mammals - primarily the human and mouse genomes. Following on from this I took a position as an SFI postdoctoral fellow in University College Cork working primarily on the evolution of genomic imprinting in mammals and plants from an analytical and quantitative perspective. Six Months later (2006) and following open competition, I moved to my first fully tenured academic post at Dublin City University. There I established my own independent research team. I have continued to expand and develop my interests in both the analytical/empirical analyses of genomic data from a molecular evolutionary perspective and also in the more theoretical concepts around evolution of life on the planet.
My research group (www.mol-evol.org) work on specific aspects of evolutionary genomics and we are primarily computational based with some wet bench collaborations/aspects to our work. We are currently 2 PhD students and 1 postdoctoral associate and have 2 further PhD positions available (see below). 
</i></p><p><strong>Contact</strong>: &nbsp;Miall 9.05,  +44(0) 113 34 34890, <img src="http://www.fbs.leeds.ac.uk/php/emailToGIF.php?ad=fbsmo" align="middle" alt="email address for " title="email address for Dr Mary O" connell'="" style="margin-bottom: 4px; display: none !important;" hidden="">&nbsp;</p><p><strong>You can read more about Dr O'Connell's interests here:</strong><br><a href="http://www.mol-evol.org/">www.mol-evol.org</a><br></p>
</div><div role="tabpanel"><ul class="nav nav-tabs" role="tablist" id="profileTab"><li role="presentation" class="active">
							<a href="http://www.fbs.leeds.ac.uk/staff/profile.php?tag=OConnell_M#research" aria-controls="research" role="tab" data-toggle="tab">Research</a>
							</li><li role="presentation" class="">
							<a href="http://www.fbs.leeds.ac.uk/staff/profile.php?tag=OConnell_M#teaching" aria-controls="teaching" role="tab" data-toggle="tab">Teaching activities</a>
							</li><li role="presentation" class="">
							<a href="http://www.fbs.leeds.ac.uk/staff/profile.php?tag=OConnell_M#group" aria-controls="group" role="tab" data-toggle="tab">Group</a>
							</li><li role="presentation" class="">
							<a href="http://www.fbs.leeds.ac.uk/staff/profile.php?tag=OConnell_M#publications" aria-controls="publications" role="tab" data-toggle="tab">Publications</a>
							</li></ul><div class="tab-content"><div role="tabpanel" class="tab-pane active" id="research"><h3>Research Interests</h3><p>Molecular Mechanisms of Evolution: Understanding the patterns and processes of evolution and the relationship between genotype, phenotype and fitness.</p><h4>Current Projects</h4><p>www.mol-evol.org</p>
&nbsp;<h3>Faculty Research and Innovation</h3><ul><li><a href="http://www.fbs.leeds.ac.uk/research/leedsomics.php">Leeds Omics</a></li><li><a href="http://www.fbs.leeds.ac.uk/research/hdd.php">Heredity, Development and Disease</a></li><li><a href="http://www.fbs.leeds.ac.uk/research/ecolevo.php">Ecology and Evolution</a></li></ul><br><hr><h3>Studentship information</h3>
<h4>Undergraduate project topics:</h4><ul><li>None </li></ul>
<h4>Postgraduate studentship areas:</h4><ul><li>Currently recruiting 2 PhD students on the following projects:</li><li>
"Genomic Innovation and the Emergence of Placentation in Mammals" http://www.findaphd.com/search/ProjectDetails.aspx?PJID=68382&amp;LID=735</li><li>
"Gene Fusion and the animal tree of life" - link to follow shortly.</li><li>
Email me at m.oconnell@leeds.ac.uk for further details. </li></ul>
<h4>See also:</h4><ul><li><a href="http://www.fbs.leeds.ac.uk/postgraduate/">Faculty Graduate School</a></li></ul></div><div role="tabpanel" class="tab-pane " id="teaching"><h4>Modules taught</h4><p><strong>BLGY1124/1128</strong> - The Diversity of Life/Living Planet<br><strong>BLGY1303</strong> - Tutorials for Biology and Genetics<br><strong>BLGY2100</strong> - Enhanced Study Skills for Biologists<br><strong>BLGY3021</strong> - Research Project<br><strong>BLGY3243</strong> - Advanced Topics in Zoology<br><strong>BLGY3291</strong> - Comparative Genomics<br><strong>BLGY3340</strong> - Biology Research Projects<br></p></div><div role="tabpanel" class="tab-pane " id="group"><h3>Postgraduates</h3><p><a href="http://www.fbs.leeds.ac.uk/staff/profile.php?un=bspm">Peter Mulhair</a> (Primary supervisor)&nbsp;90% FTE<br><a href="http://www.fbs.leeds.ac.uk/staff/profile.php?un=bsasta">Ali Taylor</a> (Primary supervisor)&nbsp;70% FTE<br><a href="http://www.fbs.leeds.ac.uk/staff/profile.php?un=bs16gn">Georgios Nikolopoulos</a> (Co-supervisor)&nbsp;33% FTE<br><a href="http://www.fbs.leeds.ac.uk/staff/profile.php?un=bsdo">David Orr</a> (Co-supervisor)&nbsp;50% FTE<br><a href="http://www.fbs.leeds.ac.uk/staff/profile.php?un=bsit">Ioannis Tsagakis</a> (Co-supervisor)&nbsp;25% FTE<br></p></div><div role="tabpanel" class="tab-pane " id="publications"><p>
		<strong>Morgan CC, Loughran NB, Walsh TA, Harrison AJ, O'Connell MJ</strong> Erratum to: Positive selection neighboring functionally essential sites and disease-implicated regions of mammalian reproductive proteins. <em>BMC evolutionary biology</em> <strong>17</strong> 170-, 2017<br><a href="http://dx.doi.org/10.1186/s12862-017-1015-y" target="_new">DOI:10.1186/s12862-017-1015-y</a></p><p>
		<strong>Lynch M, Walsh TA, Marszalowska I, Webb AE, Mac Aogáin M, Rogers TR, Windle H, Kelleher D, O’Connell MJ, Loscher CE</strong> Erratum to: Surface layer proteins from virulent Clostridium difficile ribotypes exhibit signatures of positive selection with consequences for innate immune response <em>BMC Evolutionary Biology</em> <strong>17</strong> -, 2017<br><a href="http://dx.doi.org/10.1186/s12862-017-0990-3" target="_new">DOI:10.1186/s12862-017-0990-3</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508689990&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508689990">
			<p><em>“Upon publication of the original article [1], it was noticed that there was an error in the author name. The author’s name should be "Micheál Mac Aogáin" instead of Micheál MacAogain.”</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508689990&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>Webb AE, Walsh TA, O'Connell MJ</strong> VESPA: Very large-scale Evolutionary and Selective Pressure Analyses <em>PeerJ Computer Science</em> <strong>3</strong> -, 2017<br><a href="http://dx.doi.org/10.7717/peerj-cs.118" target="_new">DOI:10.7717/peerj-cs.118</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508689991&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508689991">
			<p><em>Background: Large-scale molecular evolutionary analyses of protein coding sequences requires a number of preparatory inter-related steps from finding gene families, to generating alignments and phylogenetic trees and assessing selective pressure variation. Each phase of these analyses can represent significant challenges, particularly when working with entire proteomes (all protein coding sequences in a genome) from a large number of species. Methods: We present VESPA, software capable of automating a selective pressure analysis using codeML in addition to the preparatory analyses and summary statistics. VESPA is written in python and Perl and is designed to run within a UNIX environment. Results: We have benchmarked VESPA and our results show that the method is consistent, performs well on both large scale and smaller scale datasets, and produces results in line with previously published datasets. Discussion: Large-scale gene family identification, sequence alignment, and phylogeny reconstruction are all important aspects of large-scale molecular evolutionary analyses. VESPA provides flexible software for simplifying these processes along with downstream selective pressure variation analyses. The software automatically interprets results from codeML and produces simplified summary files to assist the user in better understanding the results.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508689991&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>McInerney JO, McNally A, O'Connell MJ</strong> Why prokaryotes have pangenomes <em>NATURE MICROBIOLOGY</em> <strong>2</strong> -, 2017<br><a href="http://dx.doi.org/10.1038/nmicrobiol.2017.40" target="_new">DOI:10.1038/nmicrobiol.2017.40</a></p><p>
		<strong>Lynch M, Walsh TA, Marszalowska I, Webb AE, MacAogain M, Rogers TR, Windle H, Kelleher D, O'Connell MJ, Loscher CE</strong> Surface layer proteins from virulent Clostridium difficile ribotypes exhibit signatures of positive selection with consequences for innate immune response <em>BMC EVOLUTIONARY BIOLOGY</em> <strong>17</strong> -, 2017<br><a href="http://dx.doi.org/10.1186/s12862-017-0937-8" target="_new">DOI:10.1186/s12862-017-0937-8</a></p><p>
		<strong>McInerney JO, O'Connell MJ</strong> Microbiology: Mind the gaps in cellular evolution <em>Nature</em> <strong>541</strong> 297-299, 2017<br><a href="http://dx.doi.org/10.1038/nature21113" target="_new">DOI:10.1038/nature21113</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508689992&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508689992">
			<p><em>Eukaryotic cells, with complex features such as membrane-bound nuclei, evolved from prokaryotic cells that lack these components. A newly identified prokaryotic group reveals intermediate steps in eukaryotic-cell evolution.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508689992&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>Williams JM, Bezak T, Das M, Ning Z, Lucking EF, Kelly VP, Harrison P, Young P, O'Connell MJ, Dockery P, O'Halloran KD, Moore T</strong> Psg22 null mouse embryos develop normally under normoxic and hypoxic conditions of pregnancy <em>Science Matters</em> <strong></strong> -, 2016<br><a href="http://dx.doi.org/10.19185/matters.201611000023" target="_new">DOI:10.19185/matters.201611000023</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508689993&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508689993">
			<p><em>Pregnancy-specific glycoproteins are secreted immunoglobulin superfamily members encoded by multigene families in eutherian mammals with haemochorial placentation. They are expressed predominantly in placental trophoblast and exhibits immunomodulatory, anti-platelet and pro-angiogenic functions. An inversion of Psg22 in the mouse locus is associated with relatively high Psg22 expression in the first half of the pregnancy. Bioinformatic analysis of seventeen mouse strains indicated that the Psg22 inversion arose at least 1.7 MYA. We used CRISPR-Cas9 mutagenesis to generate Psg22 null mutants, two of which were analysed in detail (Psg22Δ10 and Psg22Δ16). Both mutants contain frame-shifting deletions in exon 2, resulting in premature stop codons, and Psg22 mRNA was virtually undetectable. Both mutants are fertile and there was no distortion of Mendelian ratios in heterozygous crosses. Housing of pregnant females in a hypoxic (11%O2) environment for five (E5 - E10) or ten (E5 - E15) days did not induce differential growth or survival of Psg22 wildtype and null mutant genotypes. Our results indicate that Psg22 is dispensable for embryonic development and reproduction under laboratory conditions. As PSGs are secreted into maternal blood, future work will focus on whether Psg22 deficiency alters maternal physiology.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508689993&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>McRae KM, Good B, Hanrahan JP, McCabe MS, Cormican P, Sweeney T, O'Connell MJ, Keane OM</strong> Transcriptional profiling of the ovine abomasal lymph node reveals a role for timing of the immune response in gastrointestinal nematode resistance. <em>Veterinary parasitology</em> <strong>224</strong> 96-108, 2016<br><a href="http://dx.doi.org/10.1016/j.vetpar.2016.05.014" target="_new">DOI:10.1016/j.vetpar.2016.05.014</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508689994&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508689994">
			<p><em>Gastrointestinal nematodes are a serious cause of morbidity and mortality in grazing ruminants. The major ovine defence mechanism is acquired immunity, with protective immunity developing over time in response to infection. Nematode resistance varies both within and between breeds and is moderately heritable. A detailed understanding of the genes and mechanisms involved in protective immunity, and the factors that regulate this response, is required to aid both future breeding strategies and the development of effective and sustainable nematode control methods. The aim of this study was to compare the abomasal lymph node transcriptome of resistant and susceptible lambs in order to determine biological processes differentially expressed between resistant and susceptible individuals. Scottish Blackface lambs, with divergent phenotypes for resistance, were challenged with 30,000 Teladorsagia circumcincta larvae (L3), and abomasal lymph nodes recovered at 7 and 14days post-infection (dpi). High-throughput sequencing of cDNA from the abomasal lymph node was used to quantitatively sample the transcriptome with an average of 32 million reads per sample. A total of 194 and 144 genes were differentially expressed between resistant and susceptible lambs at 7 and 14 dpi respectively. Differentially expressed networks and biological processes were identified using Ingenuity Pathway Analysis. Genes involved in the inflammatory response, attraction of T lymphocytes and binding of leukocytes were more highly expressed in resistant animals at 7 dpi and in susceptible animals at 14 dpi indicating that resistant animals respond to infection earlier than susceptible animals. Twenty-four Single Nucleotide Polymorphisms (SNP) within 11 differentially expressed genes, were tested for association with gastrointestinal nematode resistance in the Scottish Blackface lambs. Four SNP, in 2 genes (SLC30A2 and ALB), were suggestively associated with faecal egg count. In conclusion, a large number of genes were differentially expressed in the abomasal lymph node of resistant and susceptible lambs responding to gastrointestinal nematode challenge. Resistant Scottish Blackface lambs appear to generate an earlier immune response to T. circumcincta. In susceptible lambs this response appears to be delayed. SNP in 2 differentially expressed genes were suggestively associated with faecal egg count indicating that differentially expressed genes may be considered candidate loci for mediating nematode resistance.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508689994&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>Tarver JE, dos Reis M, Mirarab S, Moran RJ, Parker S, O'Reilly JE, King BL, O'Connell MJ, Asher RJ, Warnow T, Peterson KJ, Donoghue PCJ, Pisani D</strong> The Interrelationships of Placental Mammals and the Limits of Phylogenetic Inference <em>Genome Biology and Evolution</em> <strong>8</strong> 330-344, 2016<br><a href="http://dx.doi.org/10.1093/gbe/evv261" target="_new">DOI:10.1093/gbe/evv261</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508689995&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508689995">
			<p><em>Placental mammals comprise three principal clades: Afrotheria (e.g., elephants and tenrecs), Xenarthra (e.g., armadillos and sloths), and Boreoeutheria (all other placental mammals), the relationships among which are the subject of controversy and a touchstone for debate on the limits of phylogenetic inference. Previous analyses have found support for all three hypotheses, leading some to conclude that this phylogenetic problem might be impossible to resolve due to the compounded effects of incomplete lineage sorting (ILS) and a rapid radiation. Here we show, using a genome scale nucleotide data set, microRNAs, and the reanalysis of the three largest previously published amino acid data sets, that the root of Placentalia lies between Atlantogenata and Boreoeutheria. Although we found evidence for ILS in early placental evolution, we are able to reject previous conclusions that the placental root is a hard polytomy that cannot be resolved. Reanalyses of previous data sets recover Atlantogenata + Boreoeutheria and show that contradictory results are a consequence of poorly fitting evolutionary models; instead, when the evolutionary process is better-modeled, all data sets converge on Atlantogenata. Our Bayesian molecular clock analysis estimates that marsupials diverged from placentals 157–170 Ma, crown Placentalia diverged 86–100 Ma, and crown Atlantogenata diverged 84–97 Ma. Our results are compatible with placental diversification being driven by dispersal rather than vicariance mechanisms, postdating early phases in the protracted opening of the Atlantic Ocean.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508689995&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>Gunn L, Collins PJ, O'Connell MJ, O'shea H</strong> Phylogenetic investigation of enteric bovine coronavirus in Ireland reveals partitioning between European and global strains <em>Irish Veterinary Journal</em> <strong>68</strong> -, 2015<br><a href="http://dx.doi.org/10.1186/s13620-015-0060-3" target="_new">DOI:10.1186/s13620-015-0060-3</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508689996&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508689996">
			<p><em>Background: Bovine coronavirus is a primary cause of neonatal calf diarrhea worldwide, and is also associated with acute diarrhea in adult cattle during the winter season. There are no reports on molecular characterization of bovine coronavirus in Ireland, and little data exists apart from serological studies. Findings: In this study, 11 neonatal (mean age 9 days) calf BCoV strains from the south of Ireland were collected over a one year period and characterized using molecular methods. The spike gene which encodes a protein involved in viral entry, infectivity and immune response shows the most variability amongst the isolates and was subsequently selected for in depth analysis. Phylogenetic analysis of the spike gene revealed that the Irish strains clustered with novel BCoV strains from Europe in a unique clade, possibly indicating lineage partitioning. Direct analysis of alignments identified amino acid changes in the spike protein unique to the Irish clade. Conclusion: Thus, monitoring of bovine coronavirus in Ireland is important as the current isolates in circulation in the south of Ireland may be diverging from the available vaccine strain, which may have implications regarding future BCoV vaccine efficacy.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508689996&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>McInerney J, Pisani D, O Connell MJ</strong> The ring of life hypothesis for eukaryote origins is supported by multiple kinds of data <em>Philosophical Transactions of the Royal Society B: Biological Sciences</em> <strong>370</strong> -, 2015<br><a href="http://dx.doi.org/10.1098/rstb.2014.0322" target="_new">DOI:10.1098/rstb.2014.0322</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508689997&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508689997">
			<p><em>© 2015 The Authors. The literature is replete with manuscripts describing the origin of eukaryotic cells. Most of the models for eukaryogenesis are either autogenous (sometimes called slow-drip), or symbiogenic (sometimes called big-bang). In this article, we use large and diverse suites of ‘Omics’ and other data to make the inference that autogeneous hypotheses are a very poor fit to the data and the origin of eukaryotic cells occurred in a single symbiosis.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508689997&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>Hughes L, Carton R, Minguzzi S, McEntee G, Deinum EE, O'Connell MJ, Parle-McDermott A</strong> An active second dihydrofolate reductase enzyme is not a feature of rat and mouse, but they do have activity in their mitochondria <em>FEBS Letters</em> <strong>589</strong> 1855-1862, 2015<br><a href="http://dx.doi.org/10.1016/j.febslet.2015.05.017" target="_new">DOI:10.1016/j.febslet.2015.05.017</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508689998&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508689998">
			<p><em>The identification of a second functional dihydrofolate reductase enzyme in humans, DHFRL1, led us to consider whether this is also a feature of rodents. We demonstrate that dihydrofolate reductase activity is also a feature of the mitochondria in both rat and mouse but this is not due to a second enzyme. While our phylogenetic analysis revealed that RNA-mediated DHFR duplication events did occur across the mammal tree, the duplicates in brown rat and mouse are likely to be processed pseudogenes. Humans have evolved the need for two separate enzymes while laboratory rats and mice have just one.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508689998&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>Webb AE, Gerek ZN, Morgan CC, Walsh TA, Loscher CE, Edwards SV, O'Connell MJ</strong> Adaptive Evolution as a Predictor of Species-Specific Innate Immune Response <em>Molecular Biology and Evolution</em> <strong>32</strong> 1717-1729, 2015<br><a href="http://dx.doi.org/10.1093/molbev/msv051" target="_new">DOI:10.1093/molbev/msv051</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508689999&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508689999">
			<p><em>It has been proposed that positive selection may be associated with protein functional change. For example, human and macaque have different outcomes to HIV infection and it has been shown that residues under positive selection in the macaque TRIM5α receptor locate to the region known to influence species-specific response to HIV. In general, however, the relationship between sequence and function has proven difficult to fully elucidate, and it is the role of large-scale studies to help bridge this gap in our understanding by revealing majorpatterns in the data that correlate genotype with function or phenotype. In this study, we investigate the level of species-specific positive selection in innate immune genes from human and mouse. In total, we analyzed 456 innate immune genes using codon-based models of evolution, comparing human,mouse, and 19 other vertebrate species to identify putative species-specific positive selection. Then we used population genomic data from the recently completed Neanderthal genome project, the 1000 human genomes project, and the 17 laboratory mouse genomes project to determine whether the residuesthat were putatively positively selected are fixed or variable in these populations. We find evidence of species-specific positive selection on both the human and the mouse branches and we show that the classes of genes under positive selection cluster by function and by interaction. Data from thisstudy provide us with targets to test the relationship between positive selection and protein function and ultimately to test the relationship between positive selection and discordant phenotypes.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508689999&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>Keane M, Semeiks J, Webb AE, Li YI, Quesada V, Craig T, Madsen LB, Van Dam S, Brawand D, Marques PI, Michalak P, Kang L, Bhak J, Yim H-S, Grishin NV, Nielsen NH, Heide-Jørgensen MP, Oziolor EM, Matson CW, Church GM, Stuart GW, Patton JC, George JC, Suydam R, Larsen K, López-Otín C, O'Connell MJ, Bickham JW, Thomsen B, De Magalhães JP</strong> Insights into the evolution of longevity from the bowhead whale genome <em>Cell Reports</em> <strong>10</strong> 112-122, 2015<br><a href="http://dx.doi.org/10.1016/j.celrep.2014.12.008" target="_new">DOI:10.1016/j.celrep.2014.12.008</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690000&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690000">
			<p><em>The bowhead whale (Balaena mysticetus) is estimated to live over 200 years and is possibly the longest-living mammal. These animals should possess protective molecular adaptations relevant to age-related diseases, particularly cancer. Here, we report the sequencing and comparative analysis of the bowhead whale genome and two transcriptomes from different populations. Our analysis identifies genes under positive selection and bowhead-specific mutations in genes linked to cancer and aging. In addition, we identify gene gain and loss involving genes associated with DNA repair, cell-cycle regulation, cancer, and aging. Our results expand our understanding of the evolution of mammalian longevity and suggest possible players involved in adaptive genetic changes conferring cancer resistance. We also found potentially relevant changes in genes related to additional processes, including thermoregulation, sensory perception, dietary adaptations, and immune response. Our data are made available online (http://www.bowhead-whale.org) to facilitate research in this long-lived species.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690000&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>O'Connell MJ, Moran RJ, Morgan CC</strong> A guide to phylogenetic reconstruction using heterogeneous models– a case study from the root of the placental mammal tree <em>Computation</em> <strong></strong> 177-196, 2015</p><p>
		<strong>McInerney JO, O'Connell MJ</strong> Evolutionary developmental biology: Ghost locus appears <em>Nature</em> <strong>514</strong> 570-571, 2014<br><a href="http://dx.doi.org/10.1038/514570a" target="_new">DOI:10.1038/514570a</a></p><p>
		<strong>Baldwin MW, Toda Y, Nakagita T, O'Connell MJ, Klasing KC, Misaka T, Edwards SV, Liberles SD</strong> Evolution of sweet taste perception in hummingbirds by transformation of the ancestral umami receptor <em>Science</em> <strong>345</strong> 929-933, 2014<br><a href="http://dx.doi.org/10.1126/science.1255097" target="_new">DOI:10.1126/science.1255097</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690001&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690001">
			<p><em>Sensory systems define an animal's capacity for perception and can evolve to promote survival in new environmental niches. We have uncovered a noncanonical mechanism for sweet taste perception that evolved in hummingbirds since their divergence from insectivorous swifts, their closest relatives. We observed the widespread absence in birds of an essential subunit (T1R2) of the only known vertebrate sweet receptor, raising questions about how specialized nectar feeders such as hummingbirds sense sugars. Receptor expression studies revealed that the ancestral umami receptor (the T1R1-T1R3 heterodimer) was repurposed in hummingbirds to function as a carbohydrate receptor. Furthermore, the molecular recognition properties of T1R1-T1R3 guided taste behavior in captive and wild hummingbirds. We propose that changing taste receptor function enabled hummingbirds to perceive and use nectar, facilitating the massive radiation of hummingbird species.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690001&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>Liu S, Lorenzen ED, Fumagalli M, Li B, Harris K, Xiong Z, Zhou L, Korneliussen TS, Somel M, Babbitt C, Wray G, Li J, He W, Wang Z, Fu W, Xiang X, Morgan CC, Doherty A, O'Connell MJ, McInerney JO, Born EW, Dalén L, Dietz R, Orlando L, Sonne C, Zhang G, Nielsen R, Willerslev E, Wang J</strong> Population genomics reveal recent speciation and rapid evolutionary adaptation in polar bears <em>Cell</em> <strong>157</strong> 785-794, 2014<br><a href="http://dx.doi.org/10.1016/j.cell.2014.03.054" target="_new">DOI:10.1016/j.cell.2014.03.054</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690002&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690002">
			<p><em>Polar bears are uniquely adapted to life in the High Arctic and have undergone drastic physiological changes in response to Arctic climates and a hyperlipid diet of primarily marine mammal prey. We analyzed 89 complete genomes of polar bear and brown bear using population genomic modeling and show that the species diverged only 479-343 thousand years BP. We find that genes on the polar bear lineage have been under stronger positive selection than in brown bears; nine of the top 16 genes under strong positive selection are associated with cardiomyopathy and vascular disease, implying important reorganization of the cardiovascular system. One of the genes showing the strongest evidence of selection, APOB, encodes the primary lipoprotein component of low-density lipoprotein (LDL); functional mutations in APOB may explain how polar bears are able to cope with life-long elevated LDL levels that are associated with high risk of heart disease in humans. PaperClip© 2014 Elsevier Inc.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690002&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>Haggerty LS, Jachiet P-A, Hanage WP, Fitzpatrick DA, Lopez P, O'Connell MJ, Pisani D, Wilkinson M, Bapteste E, McInerney JO</strong> A pluralistic account of homology: Adapting the models to the data <em>Molecular Biology and Evolution</em> <strong>31</strong> 501-516, 2014<br><a href="http://dx.doi.org/10.1093/molbev/mst228" target="_new">DOI:10.1093/molbev/mst228</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690003&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690003">
			<p><em>Defining homologous genes is important in many evolutionary studies but raises obvious issues. Some of these issues are conceptual and stem from our assumptions of how a gene evolves, others are practical, and depend on the algorithmic decisions implemented in existing software. Therefore, to make progress in the study of homology, both ontological and epistemological questions must be considered. In particular, defining homologous genes cannot be solely addressed under the classic assumptions of strong tree thinking, according to which genes evolve in a strictly tree-like fashion of vertical descent and divergence and the problems of homology detection are primarily methodological. Gene homology could also be considered under a different perspective where genes evolve as“public goods,” subjected to various introgressive processes. In this latter case, defining homologous genes becomes a matter of designing models suited to the actual complexity of the data and how such complexity arises, rather than trying to fit genetic data to some a priori tree-like evolutionary model, a practice that inevitably results in the loss of much information. Here we show how important aspects of the problems raised by homology detection methods can be overcome when even more fundamental roots of these problems are addressed by analyzing public goods thinking evolutionary processes through which genes have frequently originated. This kind of thinking acknowledges distinct types of homologs, characterized by distinct patterns, in phylogenetic and nonphylogenetic unrooted or multirooted networks. In addition, we define “family resemblances” to include genes that are related through intermediate relatives, thereby placing notions of homology in the broader context of evolutionary relationships. We conclude by presenting some payoffs of adopting such a pluralistic account of homology and family relationship, which expands the scope of evolutionary analyses beyondthe traditional, yet relatively narrow focus allowed by a strong tree-thinking view on gene evolution.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690003&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>McInerney JO, O'Connell MJ, Pisani D</strong> The hybrid nature of the Eukaryota and a consilient view of life on Earth <em>Nature Reviews Microbiology</em> <strong>12</strong> 449-455, 2014<br><a href="http://dx.doi.org/10.1038/nrmicro3271" target="_new">DOI:10.1038/nrmicro3271</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690004&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690004">
			<p><em>The origin of the eukaryotic cell, which is known as eukaryogenesis, has puzzled scientists for more than 100 years, and many hypotheses have been proposed. Recent analyses of new data enable the safe elimination of some of these hypotheses, whereas support for other hypotheses has increased. In this Opinion article, we evaluate the available theories for their compatibility with empirical observations and conclude that cellular life consists of two primary, paraphyletic prokaryotic groups and one secondary, monophyletic group that has symbiogenic origins-the eukaryotes.© 2014 Macmillan Publishers Limited. All rights reserved.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690004&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>Loughran NB, O'Connell MJ, O'Connor B,Ó'Fágáin C</strong> Stability properties of an ancient plant peroxidase <em>Biochimie</em> <strong>104</strong> 156-159, 2014<br><a href="http://dx.doi.org/10.1016/j.biochi.2014.05.012" target="_new">DOI:10.1016/j.biochi.2014.05.012</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690005&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690005">
			<p><em>Plant (Class III) peroxidases have numerous applications throughout biotechnology but their thermal and oxidative stabilities may limit their usefulness. Horseradish peroxidase isoenzyme C (HRPC) has good catalytic turnover and is moderately resistant to heat and to excess (oxidizing) concentrations of hydrogen peroxide. In contrast, HRP isoenzyme A2 (HRP A2) has better oxidative but poorer thermal stability, while soybean peroxidase (SBP) displays enhanced thermal stability. Intrigued by these variations amongst closely related enzymes, we previously used maximum likelihood methods (with application of Bayesian statistics) to infer an amino acid sequence consistent with their most recent common ancestor, the 'Grandparent' (GP). Here, we report the cloning and expression of active recombinant GP protein in Escherichia coli. GP's half-inactivation temperature was 45°C, notably less than HRP C's, but its resistance to excess H 2 O 2  was 2-fold greater. This resurrected GP protein enables a greater understanding of plant peroxidase evolution and serves as a test-bed to explore their ancestral properties. © 2014 Elsevier Masson SAS. All rights reserved.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690005&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>McRae KM, Good B, Hanrahan JP, Glynn A, O'Connell MJ, Keane OM</strong> Response to Teladorsagia circumcincta infection in Scottish Blackface lambs with divergent phenotypes for nematode resistance <em>Veterinary Parasitology</em> <strong>206</strong> 200-207, 2014<br><a href="http://dx.doi.org/10.1016/j.vetpar.2014.10.023" target="_new">DOI:10.1016/j.vetpar.2014.10.023</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690006&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690006">
			<p><em>© 2014 Elsevier B.V. The objective of this study was to identify Scottish Blackface lambs that were at the extremes of the spectrum of resistance to gastrointestinal nematodes and characterise their response to an experimental nematode challenge. Lambs (n = 90) were monitored for faecal egg count (FEC) (2 samples from each of 2 independent natural infections). The most resistant (n = 10) and susceptible (n = 10) individuals were selected and challenged with 30,000 Teladorsagia circumcincta larvae (L3) at 9 months of age. Response to infection was monitored by measuring FEC, plasma pepsinogen,serum antibodies against nematode larval antigens and haematology profile, until necropsy at 71 days post infection. Worm burden, worm fecundity and the level of anti-nematode antibodies in abomasal mucosa were determined at necropsy. FEC was consistently higher in susceptible animals (P&lt;0.05), validating the selection method. Worm fecundity was significantly reduced in resistant animals (P = 0.03). There was also a significant correlation (r = 0.88; P&lt;0.001) between the number of adult worms and FEC at slaughter. There was no effect of phenotype (resistance/susceptibility) on plasma pepsinogen or on haematology profile. Phenotype had a significant effect on the level of anti-nematode IgA antibodies in serum (P&lt;0.01), reflecting a higher peak in resistant animals at day 7 post infection.It is concluded that significant variation in the response to gastrointestinal nematode challenge exists within the Scottish Blackface population with resistant animals displaying significantly lower FEC, lower worm fecundity and higher concentration of anti-nematode IgA antibodies in serum.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690006&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>Morgan CC, Creevey CJ, O Connell MJ</strong> Mitochondrial data are not suitable for resolving placental mammal phylogeny <em>Mammalian Genome</em> <strong>25</strong> 636-647, 2014<br><a href="http://dx.doi.org/10.1007/s00335-014-9544-9" target="_new">DOI:10.1007/s00335-014-9544-9</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690007&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690007">
			<p><em>© 2014, Springer Science+Business Media New York. Mitochondrial data have traditionally been used in reconstructing a variety of species phylogenies. The low rates of recombination and thorough characterization of mitochondrial data across vertebrate species make it a particularly attractive phylogenetic marker. The relatively low number of fully sequenced mammal genomes and the lack of extensive sampling within Superorders have posed a serious problem for reaching agreement on the placement mammal species. The use of mitochondrial data sequences from large numbers of mammals could serve to circumvent the taxon-sampling deficit. Here we assess the suitability of mitochondrial data as a phylogenetic marker in mammal phylogenetics. MtDNA datasets of mammal origin have been filtered as follows: (i) we have sampled sparsely across the phylogenetic tree, (ii) we have constrained our samplingto genes with high taxon coverage, (iii) we have categorised rates across sites in a phylogeny independent manner and have removed fast evolving sites, and (iv), we have sampled from very shallow divergence times to reduce phylogenetic conflict. However, topologies obtained using these filters arenot consistent with previous studies and are discordant across different genes. Individual mitochondrial genes, and indeed all mitochondrial genes analysed as a supermatrix, resulted in poor resolution of the species phylogeny. Overall, our study highlights the limitations of mitochondrial data, notonly for resolving deep divergences and but also for shallow divergences in the mammal phylogeny.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690007&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>Collins LE, Lynch M, Marszalowska I, Kristek M, Rochfort K, O'Connell MJ, Windle H, Kelleher D, Loscher CE</strong> Surface Layer Proteins isolated from Clostridium difficile induce clearance responses in Macrophages <em>Microbes and Infection</em> <strong></strong> -, 2014</p><p>
		<strong>Morgan CC, Mc Cartney AM, Donoghue MTA, Loughran NB, Spillane C, Teeling EC, O'Connell MJ</strong> Molecular adaptation of telomere associated genes in mammals <em>BMC Evolutionary Biology</em> <strong>13</strong> -, 2013<br><a href="http://dx.doi.org/10.1186/1471-2148-13-251" target="_new">DOI:10.1186/1471-2148-13-251</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690008&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690008">
			<p><em>Background: Placental mammals display a huge range of life history traits, including size, longevity, metabolic rate and germ line generation time. Although a number of general trends have been proposed between these traits, there are exceptions that warrant further investigation. Species such as naked mole rat, human and certain bat species all exhibit extreme longevity with respect to body size. It has long been established that telomeres and telomere maintenance have a clear role in ageing but it has not yet been established whether there is evidence for adaptation in telomere maintenance proteins that could account for increased longevity in these species. Results: Here we carry out a molecular investigation of selective pressure variation, specifically focusing on telomere associated genes across placental mammals. In general we observe a large number of instances of positive selection acting on telomere genes. Although these signatures of selection overall are not significantly correlated with either longevity or body size we do identify positive selection in the microbat species Myotis lucifugus in functionally important regions of the telomere maintenance genes DKC1 and TERT, and in naked mole rat in the DNA repair gene BRCA1. Conclusion: These results demonstrate the multifarious selective pressures acting across the mammal phylogeny driving lineage-specific adaptations of telomere associated genes. Our results show that regardless of the longevity of a species, these proteins have evolved under positive selection thereby removing increased longevity as the single selective force driving this rapid rate of evolution. However, evidence of molecular adaptations specific to naked mole rat and Myotis lucifugus highlight functionally significant regions in genes that may alter the way in which telomeres are regulated and maintained in these longer-lived species.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690008&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>Morgan CC, Foster PG, Webb AE, Pisani D, McInerney JO, O'Connell MJ</strong> Heterogeneous models place the root of the placental mammal phylogeny <em>Molecular Biology and Evolution</em> <strong>30</strong> 2145-2156, 2013<br><a href="http://dx.doi.org/10.1093/molbev/mst117" target="_new">DOI:10.1093/molbev/mst117</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690009&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690009">
			<p><em>Heterogeneity among life traits in mammals has resulted in considerable phylogenetic conflict, particularly concerning the position of the placental root. Layered upon this are gene- and lineage-specific variation in amino acid substitution rates and compositional biases. Life trait variations that may impact upon mutational rates are longevity, metabolic rate, body size, and germ line generation time. Over the past 12 years, three main conflicting hypotheses have emerged for the placement of the placental root. These hypotheses place the Atlantogenata (common ancestor of Xenarthra plus Afrotheria), the Afrotheria, or the Xenarthra as the sister group to all other placental mammals. Model adequacy is critical for accurate tree reconstruction and by failing to account for these compositional and character exchange heterogeneities across the tree and data set, previous studies have not provided a strongly supported hypothesis for the placental root. For the first time, models that accommodate both tree and data set heterogeneity have been applied to mammal data. Here, we show the impact of accurate model assignment and the importance of data sets in accommodating model parameters while maintaining the power to reject competing hypotheses. Through these sophisticated methods, we demonstrate the importance of model adequacy, data set power and provide strong support for the Atlantogenata over other competing hypotheses for the position of the placental root.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690009&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>Dmitriev P, Barat A, Polesskaya A, O'Connell MJ, Robert T, Dessen P, Walsh TA, Lazar V, Turki A, Carnac G, Laoudj-Chenivesse D, Lipinski M, Vassetzky YS</strong> Simultaneous miRNA and mRNA transcriptome profiling of human myoblasts reveals a novel set of myogenic differentiation-associated miRNAs and their target genes <em>BMC Genomics</em> <strong>14</strong> -, 2013<br><a href="http://dx.doi.org/10.1186/1471-2164-14-265" target="_new">DOI:10.1186/1471-2164-14-265</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690010&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690010">
			<p><em>Background: miRNA profiling performed in myogenic cells and biopsies from skeletal muscles has previously identified miRNAs involved in myogenesis. Results: Here, we have performed miRNA transcriptome profiling in human affinity-purified CD56+ myoblasts induced to differentiate in vitro. In total, we have identified 60 miRNAs differentially expressed during myogenic differentiation. Many were not known for being differentially expressed during myogenic differentiation. Of these, 14 (miR-23b, miR-28, miR-98, miR-103, miR-107, miR-193a, miR-210, miR-324-5p, miR-324-3p, miR-331, miR-374, miR-432, miR-502, and miR-660) were upregulated and 6 (miR-31, miR-451, miR-452, miR-565, miR-594 and miR-659) were downregulated. mRNA transcriptome profiling performed in parallel resulted in identification of 6,616 genes differentially expressed during myogenic differentiation. Conclusions: This simultaneous miRNA/mRNA transcriptome profiling allowed us to predict with high accuracy target genes of myogenesis-related microRNAs and to deduce their functions.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690010&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>Loughran NB, Hinde S, McCormick-Hill S, Leidal KG, Bloomberg S, Loughran ST, O'Connor B,Ó'Fágáin C, Nauseef WM, O'Connell MJ</strong> Functional consequence of positive selection revealed through rational mutagenesis of human myeloperoxidase <em>Molecular Biology and Evolution</em> <strong>29</strong> 2039-2046, 2012<br><a href="http://dx.doi.org/10.1093/molbev/mss073" target="_new">DOI:10.1093/molbev/mss073</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690011&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690011">
			<p><em>Myeloperoxidase (MPO) is a member of the mammalian heme peroxidase (MHP) multigene family. Whereas all MHPs oxidize specific halides to generate the corresponding hypohalous acid, MPO is unique in its c apacity to oxidize chloride at physiologic pH to produce hypochlorous acid (HOCl), a potent microbicide that contributes to neutrophil-mediated host defense against infection. We have previously resolved the evolutionary relationships in this functionally diverse multigene family and predicted in silico that positive Darwinian selection played a major role in the observed functional diversities (Loughran NB, O'Connor B, O'Fagain C, O'Connell MJ. 2008. The phylogeny of the mammalian heme peroxidases and the evolution of their diverse functions. BMC Evol Biol. 8:101). In this work, we have replaced positively selected residues asparagine 496 (N496), tyrosine 500 (Y500), and leucine 504 (L504) with the amino acids present in the ancestral MHP and have examined the effects on the structure, biosynthesis, and activity of MPO. Analysis in silico predicted that N496F, Y500F, or L504T would perturb hydrogen bonding in the heme pocket of MPO and thus disrupt the structural integrity of the enzyme. Biosynthesis of the mutants stably expressed in human embryonic kidney 293 cells yielded apoproMPO, the heme-free, enzymatically inactive precursor of MPO, that failed to undergo normal maturation or proteolytic processing. As a consequence of the maturational arrest at the apoproMPO stage of development, cells expressing MPO with mutations N496F, Y500F, L504T, individually or in combination, lacked normal peroxidase or chlorinating activity. Taken together, our data provide further support for the in silico predictions of positive selection and highlight the correlation between positive selection and functional divergence. Our data demonstrate that directly probing the functional importance of positive selection can provide important insights into understanding protein evolution.© The Author 2012. Published by Oxford University Press on behalf of the Society for Molecular Biology and Evolution. All rights reserved.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690011&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>O'Connell MJ, Doyle AM, Juenger TE, Donoghue MTA, Keshavaiah C, Tuteja R, Spillane C</strong> In Arabidopsis thaliana codon volatility scores reflect GC3 composition rather than selective pressure <em>BMC Research Notes</em> <strong>5</strong> -, 2012<br><a href="http://dx.doi.org/10.1186/1756-0500-5-359" target="_new">DOI:10.1186/1756-0500-5-359</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690012&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690012">
			<p><em>Background: Synonymous codon usage bias has typically been correlated with, and attributed to translational efficiency. However, there are other pressures on genomic sequence composition that can affect codon usage patterns such as mutational biases. This study provides an analysis of the codon usage patterns in Arabidopsis thaliana in relation to gene expression levels, codon volatility, mutational biases and selective pressures. Results: We have performed synonymous codon usage and codon volatility analyses for all genes in the A. thaliana genome. In contrast to reports for species from other kingdoms, we find that neither codon usage nor volatility are correlated with selection pressure (as measured by dN/dS), nor with gene expression levels on a genome wide level. Our results show that codon volatility and usage are not synonymous, rather that they are correlated with the abundance of G and C at the third codon position (GC3). Conclusions: Our results indicate that while the A. thaliana genome shows evidence for synonymous codon usage bias, this is not related to the expression levels of its constituent genes. Neither codon volatility nor codon usage are correlated with expression levels or selective pressures but, because they are directly related to the composition of G and C at the third codon position, they are the result of mutational bias. Therefore, in A. thaliana codon volatility and usage do not result from selection for translation efficiency or protein functional shift as measured by positive selection.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690012&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>Shakya K, O'Connell MJ, Ruskin HJ</strong> The landscape for epigenetic/epigenomic biomedical resources <em>Epigenetics</em> <strong>7</strong> 982-986, 2012<br><a href="http://dx.doi.org/10.4161/epi.21493" target="_new">DOI:10.4161/epi.21493</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690013&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690013">
			<p><em>Recent advances in molecular biology and computational power have seen the biomedical sector enter a new era, with corresponding development of bioinformatics as a major discipline. Generation of enormous amounts of data has driven the need for more advanced storage solutions and shared access through a range of public repositories. The number of such biomedical resources is increasing constantly and mining these large and diverse data sets continues to present real challenges. This paper attempts a general overview of currently available resources, together with remarks on their data mining and analysis capabilities. Of interest here is the recent shift in focus from genetic to epigenetic/epigenomic research and the emergence and extension of resource provision to support this both at local and global scale. Biomedical text and numerical data mining are both considered, the first dealing with automated methods for analyzing research content and information extraction, and the second (broadly) with pattern recognition and prediction. Any summary and selection of resources is inherently limited, given the spectrum available, but the aim is to provide a guideline for the assessment and comparison of currently available provision, particularly as this relates to epigenetics/epigenomics.© 2012 Landes Bioscience.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690013&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>Morgan CC, Shakya K, Webb A, Walsh TA, Lynch M, Loscher CE, Ruskin HJ, O'Connell MJ</strong> Colon cancer associated genes exhibit signatures of positive selection at functionally significant positions. <em>BMC evolutionary biology</em> <strong>12</strong> 114-, 2012<br><a href="http://dx.doi.org/10.1186/1471-2148-12-114" target="_new">DOI:10.1186/1471-2148-12-114</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690014&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690014">
			<p><em>BACKGROUND: Cancer, much like most human disease, is routinely studied by utilizing model organisms. Of these model organisms, mice are often dominant. However, our assumptions of functional equivalence fail to consider the opportunity for divergence conferred by ~180 Million Years (MY) of independent evolution between these species. For a given set of human disease related genes, it is therefore important to determine if functional equivalency has been retained between species. In this study we test the hypothesis that cancer associated genes have different patterns of substitution akin to adaptive evolution in different mammal lineages. RESULTS: Our analysis of the current literature and colon cancer databases identified 22 genes exhibiting colon cancer associated germline mutations. We identified orthologs for these 22 genes across a set of high coverage (&gt;6X) vertebrate genomes. Analysis of these orthologous datasets revealed significant levels of positive selection. Evidence of lineage-specific positive selection was identified in 14 genes in both ancestral and extant lineages. Lineage-specific positive selection was detected in the ancestral Euarchontoglires and Hominidae lineages for STK11, in the ancestral primate lineage for CDH1, in the ancestral Murinae lineage for both SDHC and MSH6 genes and the ancestral Muridae lineage for TSC1. CONCLUSION: Identifying positive selection in the Primate, Hominidae, Muridae and Murinae lineages suggests an ancestral functional shift in these genes between the rodent and primate lineages. Analyses such as this, combining evolutionary theory and predictions - along with medically relevant data, can thus provide us with important clues for modeling human diseases.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690014&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>McInerney JO, Pisani D, Bapteste E, O'Connell MJ</strong> The public goods hypothesis for the evolution of life on Earth <em>Biology Direct</em> <strong>6</strong> -, 2011<br><a href="http://dx.doi.org/10.1186/1745-6150-6-41" target="_new">DOI:10.1186/1745-6150-6-41</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690015&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690015">
			<p><em>It is becoming increasingly difficult to reconcile the observed extent of horizontal gene transfers with the central metaphor of a great tree uniting all evolving entities on the planet. In this manuscript we d escribe the Public Goods Hypothesis and show that it is appropriate in order to describe biological evolution on the planet. According to this hypothesis, nucleotide sequences (genes, promoters, exons, etc.) are simply seen as goods, passed from organism to organism through both vertical and horizontal transfer. Public goods sequences are defined by having the properties of being largely non-excludable (no organism can be effectively prevented from accessing these sequences) and non-rival (while such a sequence is being used by one organism it is also available for use by another organism). The universal nature of genetic systems ensures that such non-excludable sequences exist and non-excludability explains why we see a myriad of genes in different combinations in sequenced genomes. There are three features of the public goods hypothesis. Firstly, segments of DNA are seen as public goods, available for all organisms to integrate into their genomes. Secondly, we expect the evolution of mechanisms for DNA sharing and of defense mechanisms against DNA intrusion in genomes. Thirdly, we expect that we do not see a global tree-like pattern. Instead, we expect local tree-like patterns to emerge from the combination of a commonage of genes and vertical inheritance of genomes by cell division. Indeed, while genes are theoretically public goods, in reality, some genes are excludable, particularly, though not only, when they have variant genetic codes or behave as coalition or club goods, available for all organisms of a coalition to integrate into their genomes, and non-rival within the club. We view the Tree of Life hypothesis as a regionalized instance of the Public Goods hypothesis, just like classical mechanics and euclidean geometry are seen as regionalized instances of quantum mechanics and Riemannian geometry respectively. We argue for this change using an axiomatic approach that shows that the Public Goods hypothesis is a better accommodation of the observed data than the Tree of Life hypothesis.© 2011 McInerney et al; licensee BioMed Central Ltd.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690015&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>Ryan A, Lynch M, Smith SM, Amu S, Nel HJ, McCoy CE, Dowling JK, Draper E, O'Reilly V, McCarthy C, O'Brien J, Eidhin D, O'Connell MJ, Keogh B, Morton CO, Rogers TR, Fallon PG, O'Neill LA, Kelleher D, Loscher CE</strong> A role for TLR4 in clostridium difficile infection and the recognition of surface layer proteins <em>PLoS Pathogens</em> <strong>7</strong> -, 2011<br><a href="http://dx.doi.org/10.1371/journal.ppat.1002076" target="_new">DOI:10.1371/journal.ppat.1002076</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690016&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690016">
			<p><em>Clostridium difficile is the etiological agent of antibiotic-associated diarrhoea (AAD) and pseudomembranous colitis in humans. The role of the surface layer proteins (SLPs) in this disease has not yet ion of C. difficile and the subsequent activation of the immune system. Bone marrow derived dendritic cells (DCs) exposed to SLPs were assessed for production of inflammatory cytokines, expression of cell surface markers and their ability to generate T helper (Th) cell responses. DCs isolated from C3H/HeN and C3H/HeJ mice were used in order to examine whether SLPs are recognised by TLR4. The role of TLR4 in infection was examined in TLR4-deficient mice. SLPs induced maturation of DCs characterised by production of IL-12, TNFα and IL-10 and expression of MHC class II, CD40, CD80 and CD86. Furthermore, SLP-activated DCs generated Th cells producing IFNγ and IL-17. SLPs were unable to activate DCs isolated from TLR4-mutant C3H/HeJ mice and failed to induce a subsequent Th cell response. TLR4 -/-  and Myd88 -/- , but notTRIF -/-  mice were more susceptible than wild-type mice to C. difficile infection. Furthermore, SLPs activated NFκB, but not IRF3, downstream of TLR4. Our results indicate that SLPs isolated from C. difficile can activate innate and adaptive immunity and that these effects are mediated by TLR4, with TLR4 having a functio nal role in experimental C. difficile infection. This suggests an important role for SLPs in the recognition of C. difficile by the immune system. © 2011 Ryan et al.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690016&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>Mccole RB, Loughran NB, Chahal M, Fernandes LP, Roberts RG, Fraternali F, O'Connell MJ, Oakey RJ</strong> A Case-By-Case Evolutionary Analysis Of Four Imprinted Retrogenes <em>Evolution</em> <strong>65</strong> 1413-1427, 2011<br><a href="http://dx.doi.org/10.1111/j.1558-5646.2010.01213.x" target="_new">DOI:10.1111/j.1558-5646.2010.01213.x</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690017&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690017">
			<p><em>Retroposition is a widespread phenomenon resulting in the generation of new genes that are initially related to a parent gene via very high coding sequence similarity. We examine the evolutionary fate of four retrogenes generated by such an event; mouseInpp5f-v2, Mcts2, Nap1l5, andU2af1-rs1.These genes are all subject to the epigenetic phenomenon of parental imprinting. We first provide new data on the age of these retrogene insertions. Using codon-based models of sequence evolution, we show these retrogenes have diverse evolutionary trajectories, including divergence from the parent coding sequence under positive selection pressure, purifying selection pressure maintaining parent-retrogene similarity, and neutral evolution. Examination of the expression pattern of retrogenes shows an atypical, broad pattern across multiple tissues. Protein 3D structure modeling reveals that a positively selected residue inU2af1-rs1, not shared by its parent, may influence protein conformation. Our case-by-case analysis of the evolution of four imprinted retrogenes reveals that this interesting class of imprinted genes, while similar in regulation and sequence characteristics, follow very varied evolutionary paths.© 2011 The Author(s). Evolution© 2011 The Society for the Study of Evolution.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690017&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>O'Connell MJ</strong> Selection and the cell cycle: Positive darwinian selection in a well-Known DNA damage response pathway <em>Journal of Molecular Evolution</em> <strong>71</strong> 444-457, 2010<br><a href="http://dx.doi.org/10.1007/s00239-010-9399-y" target="_new">DOI:10.1007/s00239-010-9399-y</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690018&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690018">
			<p><em>Cancer is a common occurrence in multi-cellular organisms and is not strictly limited to the elderly in a population. It is therefore possible that individuals with genotypes that protect against early onset cancers have a selective advantage. In this study the patterns of mutation in the proteins of a well-studied DNA damage response pathway have been examined for evidence of adaptive evolutionary change. Using a maximum likelihood framework and the mammalian species phylogeny, together with codon models of evolution, selective pressure variation across the interacting network of proteins has been detected. The presence of signatures of adaptive evolution in BRCA1 and BRCA2 has already been documented but the effect on the entire network of interacting proteins in this damage response pathway has, until now, been unknown. Positive selection is evident throughout the network with a total of 11 proteins out of 15 examined displaying patterns of substitution characteristic of positive selection. It is also shown here that modern human populations display evidence of an ongoing selective sweep in 9 of these DNA damage repair proteins. The results presented here provide the community with new residues that may be relevant to cancer susceptibility while also highlighting those proteins where human and mouse have undergone lineage-specific functional shift. An understanding of this damage response pathway from an evolutionary perspective will undoubtedly contribute to future cancer treatment approaches.© 2010 Springer Science+Business Media, LLC.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690018&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>O'Connell MJ, Loughran NB, Walsh TA, Donoghue MTA, Schmid KJ, Spillane C</strong> A phylogenetic approach to test for evidence of parental conflict or gene duplications associated with protein-encoding imprinted orthologous genes in placental mammals <em>Mammalian Genome</em> <strong>21</strong> 486-498, 2010<br><a href="http://dx.doi.org/10.1007/s00335-010-9283-5" target="_new">DOI:10.1007/s00335-010-9283-5</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690019&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690019">
			<p><em>There are multiple theories on the evolution of genomic imprinting. We investigated whether the molecular evolution of true orthologs of known imprinted genes provides support for theories based on gene duplication or parental conflicts (where mediated by amino-acid changes). Our analysis of 34 orthologous genes demonstrates that the vast majority of mammalian imprinted genes have not undergone any subsequent significant gene duplication within placental species, suggesting that selection pressures against gene duplication events could be operating for imprinted loci. As antagonistic co-evolution between imprinted genes can regulate offspring growth, proteins mediating this interaction could be subject to rapid evolution via positive selection. Supporting this, we detect evidence of site specific positive selection for the imprinted genes OSBPL5 (and GNASXL), and detect lineage-specific positive selection for 14 imprinted genes where it is known that the gene is imprinted in a specific lineage, namely for: PLAGL1, IGF2, SLC22A18, OSBPL5, DCN, DLK1, RASGRF1, IGF2R, IMPACT, GRB10, NAPIL4, UBE3A, GATM and GABRG3. However, there is an overall lack of concordance between the known imprinting status of each gene (i.e. whether the gene is imprinted or biallelically expressed in a particular mammalian lineage) and positive selection. While only a small number of orthologs of imprinted loci display evidence of positive selection, we observe that the majority of orthologs of imprinted loci display high levels of micro-synteny conservation and have undergone very few cis- or trans-duplications in placental mammalian lineages.© 2010 Springer Science+Business Media, LLC.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690019&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>Morgan CC, Loughran NB, Walsh TA, Harrison AJ, O'Connell MJ</strong> Positive selection neighboring functionally essential sites and disease-implicated regions of mammalian reproductive proteins <em>BMC Evolutionary Biology</em> <strong>10</strong> -, 2010<br><a href="http://dx.doi.org/10.1186/1471-2148-10-39" target="_new">DOI:10.1186/1471-2148-10-39</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690020&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690020">
			<p><em>Background. Reproductive proteins are central to the continuation of all mammalian species. The evolution of these proteins has been greatly influenced by environmental pressures induced by pathogens, rival sperm, sexual selection and sexual conflict. Positive selection has been demonstrated in many of these proteins with particular focus on primate lineages. However, the mammalia are a diverse group in terms of mating habits, population sizes and germ line generation times. We have examined the selective pressures at work on a number of novel reproductive proteins across a wide variety of mammalia. Results. We show that selective pressures on reproductive proteins are highly varied. Of the 10 genes analyzed in detail, all contain signatures of positive selection either across specific sites or in specific lineages or a combination of both. Our analysis of SP56 and Col1a1 are entirely novel and the results show positively selected sites present in each gene. Our findings for the Col1a1 gene are suggestive of a link between positive selection and severe disease type. We find evidence in our dataset to suggest that interacting proteins are evolving in symphony: most likely to maintain interacting functionality. Conclusion. Our in silico analyses show positively selected sites are o ccurring near catalytically important regions suggesting selective pressure to maximize efficient fertilization. In those cases where a mechanism of protein function is not fully understood, the sites presented here represent ideal candidates for mutational study. This work has highlighted the widespread rate heterogeneity in mutational rates across the mammalia and specifically has shown that the evolution of reproductive proteins is highly varied depending on the species and interacting partners. We have shown that positive selection and disease are closely linked in the Col1a1 gene.© 2010 Morgan et al; licensee BioMed Central Ltd.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690020&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>Ryan BJ, O'Connell MJ,Ó'Fágáin C</strong> Consensus mutagenesis reveals that non-helical regions influence thermal stability of horseradish peroxidase <em>Biochimie</em> <strong>90</strong> 1389-1396, 2008<br><a href="http://dx.doi.org/10.1016/j.biochi.2008.04.009" target="_new">DOI:10.1016/j.biochi.2008.04.009</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690021&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690021">
			<p><em>The enzyme horseradish peroxidase has many uses in biotechnology but a stabilized derivative would have even wider applicability. To enhance thermal stability, we applied consensus mutagenesis (used successfully with other proteins) to recombinant horseradish peroxidase and generated five single-site mutants. Unexpectedly, these mutations had greater effects on steady-state kinetics than on thermal stability. Only two mutants (T102A, T110V) marginally exceeded the wild type's thermal stability (4% and 10% gain in half-life at 50°C respectively); the others (Q106R, Q107D, I180F) were less stable than wild type. Stability of a five-fold combination mutant matched that of Q106R, the least-stable single mutant. These results were perplexing: the Class III plant peroxidases display wide differences in thermal stability, yet the consensus mutations failed to reflect these natural variations. We examined the sequence content of Class III peroxidases to determine if there are identifiable molecular reasons for the stability differences observed. Bioinformatic analysis validated our choice of sites and mutations and generated an archetypal peroxidase sequence for comparison with extant sequences. It seems that both genetic variation and differences in protein stability are confined to non-helical regions due to the presence of a highly conserved alpha-helical structural scaffold in these enzymes. © 2008 Elsevier Masson SAS. All rights reserved.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690021&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>O'Connell MJ, McInerney JO</strong> Reconstructing the ancestral eukaryote: lessons from the past In <em>Ancestral Sequence Reconstruction</em> , 2008<br><a href="http://dx.doi.org/10.1093/acprof:oso/9780199299188.003.0010" target="_new">DOI:10.1093/acprof:oso/9780199299188.003.0010</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690022&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690022">
			<p><em>© Oxford University Press, 2013. Reconstructing ancestral proteins relies heavily on accurate phylogeny reconstruction. This is a challenging area particularly for early eukaryote evolution. Despite decades of extensive studies to resolve the structure of the deepest branches of the Eukaryota, a generally accepted phylogenetic reconstruction has not been achieved. This chapter explores the relationships of the three crown eukaryota and the root of the animal kingdom itself. It describes the methodological developments that have aided the field of early eukaryote evolution and the biases thatexist in certain methods and data.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690022&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>Loughran NB, O'Connor B,Ó'Fágáin C, O'Connell MJ</strong> The phylogeny of the mammalian heme peroxidases and the evolution of their diverse functions <em>BMC Evolutionary Biology</em> <strong>8</strong> -, 2008<br><a href="http://dx.doi.org/10.1186/1471-2148-8-101" target="_new">DOI:10.1186/1471-2148-8-101</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690023&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690023">
			<p><em>Background. The mammalian heme peroxidases (MHPs) are a medically important group of enzymes. Included in this group are myeloperoxidase, eosinophil peroxidase, lactoperoxidase, and thyroid peroxidase. These enzymes are associated with such diverse diseases as asthma, Alzheimer's disease and inflammatory vascular disease. Despite much effort to elucidate a clearer understanding of the function of the 4 major groups of this multigene family, we still do not have a clear understanding of their relationships to each other. Results. Sufficient signal exists for the resolution of the evolutionary relationships of this family of enzymes. We demonstrate, using a root mean squared deviation statistic, how the removal of the fastest evolving sites aids in the minimisation of the effect of long branch attraction and the generation of a highly supported phylogeny. Based on this phylogeny we have pinpointed the amino acid positions that have most likely contributed to the diverse functions of these enzymes. Many of these residues are in close proximity to sites implicated in protein misfolding, loss of function or disease. Conclusion. Our analysis of all available genomic sequence data for the MHPs from all available completed mammalian genomes, involved sophisticated methods of phylogeny reconstruction and data treatment. Our study has (i) fully resolved the phylogeny of the MHPs and the subsequent pattern of gene duplication, and (ii), we have detected amino acids under positive selection that have most likely contributed to the observed functional shifts in each type of MHP.© 2008 Loughran et al; licensee BioMed Central Ltd.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690023&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>Haun WJ, Laoueillé-Duprat S, O'Connell MJ, Spillane C, Grossniklaus U, Phillips AR, Kaeppler SM, Springer NM</strong> Genomic imprinting, methylation and molecular evolution of maize Enhancer of zeste (Mez) homologs <em>Plant Journal</em> <strong>49</strong> 325-337, 2007<br><a href="http://dx.doi.org/10.1111/j.1365-313X.2006.02965.x" target="_new">DOI:10.1111/j.1365-313X.2006.02965.x</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690024&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690024">
			<p><em>Imprinted gene expression refers to differential transcription of alleles depending on their parental origin. To date, most examples of imprinted gene expression in plants occur in the triploid endosperm tissue. The Arabidopsis gene MEDEA displays an imprinted pattern of gene expression and has homology to the Drosophila Polycomb group (PcG) protein Enhancer-of-zeste (E(z)). We have tested the allele-specific expre ssion patterns of the three maize E(z)-like genes Mez1, Mez2 and Mez3. The expression of Mez2 and Mez3 is not imprinted, with a bi-allelic pattern of transcription for both genes in both the endosperm and embryonic tissue. In contrast, Mez1 displays a bi-allelic expression pattern in the embryonic tissue, and a mono-allelic expression pattern in the developing endosperm tissue. We demonstrate that mono-allelic expression of the maternal Mez1 allele occurs throughout endosperm development. We have identified a 556 bp differentially methylated region (DMR) located approximately 700 bp 5′ of the Mez1 transcription start site. This region is heavily methylated at CpG and CpNpG nucleotides on the non-expressed paternal allele but has low levels of methylation on the expressed maternal allele. Molecular evolutionary analysis indicates that conserved domains of all three Mez genes are under purifying selection. The common imprinted expression of Mez1 and MEDEA, in concert with their likely evolutionary origins, suggests that there may be a requirement for imprinting of at least one E(z)-like gene in angiosperms. © 2007 The Authors.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690024&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>O'Connell MJ, McInerney JO</strong> Adaptive evolution of the human fatty acid synthase gene: Support for the cancer selection and fat utilization hypotheses? <em>Gene</em> <strong>360</strong> 151-159, 2005<br><a href="http://dx.doi.org/10.1016/j.gene.2005.06.020" target="_new">DOI:10.1016/j.gene.2005.06.020</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690025&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690025">
			<p><em>Cancer may act as the etiological agent for natural selection in some genes. This selective pressure would act to reduce the success of neoplastic lineages over normal cell lineages in individuals of reproductive age. In addition, human's relatively larger brain and longer lifespan may have also acted as a selective force requiring new genotypes. One of the most important proteins in both processes is the fatty acid synthase (FAS) gene involved in fatty acid biosynthesis. A variety of other proteins, including PTEN, MAPK1, SREBP1, SREBP2 and PI are also involved in the regulation of fatty acid biosynthesis. We have specifically analysed variability in selective pressure across all these genes in human, mouse and other vertebrates. We have found that the FAS gene alone has signatures indicative of adaptive evolution. We did not find any signatures of adaptive evolution in any of the other proteins. In the FAS gene, we have detected an excess of non-synonymous over synonymous substitutions in approximately 6% of sites in the human lineage. Contrastingly, the substitution process at these sites in other available vertebrates and mammals indicates strong purifying selection. This is likely to reflect a functional shift in human FAS and correlates well with previously observed changes in FAS biochemical activities. We speculate that the role played by FAS either in cancer development or in human brain development has created this selective pressure, although we cannot rule out the various other functions of FAS.© 2005 Elsevier B.V. All rights reserved.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690025&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>O'Connell MJ, McInerney JO</strong> Gamma chain receptor interleukins: Evidence for positive selection driving the evolution of cell-to-cell communicators in the mammalian immune system <em>Journal of Molecular Evolution</em> <strong>61</strong> 608-619, 2005<br><a href="http://dx.doi.org/10.1007/s00239-004-0313-3" target="_new">DOI:10.1007/s00239-004-0313-3</a><br><a href="javascript:void(0)" onclick="$(&#39;#a1508690026&#39;).slideDown(&#39;slow&#39;)">View abstract</a></p>
			<div style="display:none; margin-left:30px" id="a1508690026">
			<p><em>The interleukin-2 receptor (IL-2R)γ chain, or common γ chain (γc), is the hub of a protein interaction network in the mammalia that is central to defense against disease. It is the indispensable subunit of the functional receptor complexes for a group of interleukins known as the γ-chain-dependent interleukins (IL-2, IL-4, -7, -9, -15, and -21). The γc links these proteins through their interaction with it and their competition for its recruitment. The γc-dependent interleukins also interact with each other to either enhance or suppress expression through manipulation of expression of receptor subunits. Given the influence of protein-protein interactions on evolution, such as those documented for many genes including the reproductive proteins of the sperm and egg coat, here we have asked whether there is a common thread in the evolution of these interleukins. Our findings indicate that positive selection has actedby fixing a large number of amino acid replacement mutations in every single one of these interleukins, this adaptive evolution is also observed in a lineage-specific manner. Crucially, however, there does not appear to have ever been an instance of adaptive evolution in the γc chain itself, thereby providing an insight into the evolution of this hub protein. These findings highlight the importance of adaptive evolutionary events in the evolution of this central network in the immune system and suggest underlying causes for differences in defense responses in the mammalia. © Springer Science+Business Media, Inc. 2005.</em>
			<br><a href="javascript:void(0)" onclick="$(&#39;#a1508690026&#39;).slideUp(&#39;slow&#39;)">hide</a></p>
			</div><p>
		<strong>Travers SA, O'Connell MJ, McCormack G, McInerney JO</strong> Evidence for heterogeneous selective pressures in the evolution of the env gene in different human immunodeficiency virus type 1 subtypes <em>Journal of Virology</em> <strong></strong> -, 2005</p><p>
		<strong>Creevey CJ, Fitzpatrick DA, Philip GK, Kinsella RJ, O'Connell MJ, Pentony MM, Travers SA, Wilkinson M, McInerney JO</strong> A tree-like phylogeny only exists at the tips in the prokaryotes <em>Proceedings of the Royal Society of London Biology</em> <strong></strong> -, 2004</p></div></div></div></div></div><!-- row -->

      <div class="clearfix"></div>

      <footer class="footer">
                  <p style="float: right;"><a href="https://www.facebook.com/biologicalsciencesleeds/"><img src="./Dr Mary O&#39;Connell_files/facebook.png" alt="Facebook" title="Facebook" style="width: 30px;"></a> <a href="https://twitter.com/ScienceLeeds"><img src="./Dr Mary O&#39;Connell_files/twitter.png" alt="Twitter" title="Twitter" style="width: 30px;"></a> <a href="https://www.youtube.com/channel/UC_CptNsc1xIIeqfGwMDdxLA"><img src="./Dr Mary O&#39;Connell_files/youtube.png" alt="YouTube" title="YouTube" style="width: 30px;"></a> <a href="http://instagram.com/fbsleeds"><img src="./Dr Mary O&#39;Connell_files/instagrams.png" alt="Instagram" title="Instagram" style="width: 30px;"></a></p>
                <p><a href="http://www.fbs.leeds.ac.uk/staff/alpha.htm" rel="external">Staff A-Z</a> | <a href="http://www.fbs.leeds.ac.uk/newstaff/" rel="external">Information for new staff</a> | <a href="http://www.fbs.leeds.ac.uk/staff/contacts.htm" rel="external">Contact us</a></p>
        <p><a href="http://www.fbs.leeds.ac.uk/staff/profile.php?tag=OConnell_M#top" rel="external">Back to the top</a> | <a href="http://www.fbs.leeds.ac.uk/privacy.htm" rel="external">Privacy</a> | <a href="http://www.leeds.ac.uk/freedom-of-information" rel="external">Freedom of Information</a></p>
        <p>© 2017 University of Leeds, Leeds, LS2 9JT.   Content on this page is maintained by the <a href="mailto:websupport@fbs.leeds.ac.uk" rel="external">faculty web team</a></p>
      </footer>

      </div><!-- jumbotron -->

    </div> <!-- /container -->
    

  <script>
	// setup any tables which need to be presented as accordions
	setup_table_accordions(); // defined in bootstrap-override.js

    // Toggle accordion icon for open/close
    $("a.acclink").click(function() {
      if ($(this).find("span.glyphicon").hasClass("glyphicon-plus-sign")) {
        $(this).find("span.glyphicon").addClass("glyphicon-minus-sign").removeClass("glyphicon-plus-sign");
      } else if ($(this).find("span.glyphicon").hasClass("glyphicon-minus-sign")) {
        $(this).find("span.glyphicon").addClass("glyphicon-plus-sign").removeClass("glyphicon-minus-sign");
      }
    });
    
      </script>

  
<!-- end -->


<script>
$(function() {
    $('#profileTab a').click(function (e) {
        e.preventDefault();
        $(this).tab('show');
    })
});
</script><span style="border-radius: 3px; text-indent: 20px; width: auto; padding: 0px 4px 0px 0px; text-align: center; font-style: normal; font-variant: normal; font-weight: bold; font-stretch: normal; font-size: 11px; line-height: 20px; font-family: &quot;Helvetica Neue&quot;, Helvetica, sans-serif; color: rgb(255, 255, 255); background: url(&quot;data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMzBweCIgd2lkdGg9IjMwcHgiIHZpZXdCb3g9Ii0xIC0xIDMxIDMxIj48Zz48cGF0aCBkPSJNMjkuNDQ5LDE0LjY2MiBDMjkuNDQ5LDIyLjcyMiAyMi44NjgsMjkuMjU2IDE0Ljc1LDI5LjI1NiBDNi42MzIsMjkuMjU2IDAuMDUxLDIyLjcyMiAwLjA1MSwxNC42NjIgQzAuMDUxLDYuNjAxIDYuNjMyLDAuMDY3IDE0Ljc1LDAuMDY3IEMyMi44NjgsMC4wNjcgMjkuNDQ5LDYuNjAxIDI5LjQ0OSwxNC42NjIiIGZpbGw9IiNmZmYiIHN0cm9rZT0iI2ZmZiIgc3Ryb2tlLXdpZHRoPSIxIj48L3BhdGg+PHBhdGggZD0iTTE0LjczMywxLjY4NiBDNy41MTYsMS42ODYgMS42NjUsNy40OTUgMS42NjUsMTQuNjYyIEMxLjY2NSwyMC4xNTkgNS4xMDksMjQuODU0IDkuOTcsMjYuNzQ0IEM5Ljg1NiwyNS43MTggOS43NTMsMjQuMTQzIDEwLjAxNiwyMy4wMjIgQzEwLjI1MywyMi4wMSAxMS41NDgsMTYuNTcyIDExLjU0OCwxNi41NzIgQzExLjU0OCwxNi41NzIgMTEuMTU3LDE1Ljc5NSAxMS4xNTcsMTQuNjQ2IEMxMS4xNTcsMTIuODQyIDEyLjIxMSwxMS40OTUgMTMuNTIyLDExLjQ5NSBDMTQuNjM3LDExLjQ5NSAxNS4xNzUsMTIuMzI2IDE1LjE3NSwxMy4zMjMgQzE1LjE3NSwxNC40MzYgMTQuNDYyLDE2LjEgMTQuMDkzLDE3LjY0MyBDMTMuNzg1LDE4LjkzNSAxNC43NDUsMTkuOTg4IDE2LjAyOCwxOS45ODggQzE4LjM1MSwxOS45ODggMjAuMTM2LDE3LjU1NiAyMC4xMzYsMTQuMDQ2IEMyMC4xMzYsMTAuOTM5IDE3Ljg4OCw4Ljc2NyAxNC42NzgsOC43NjcgQzEwLjk1OSw4Ljc2NyA4Ljc3NywxMS41MzYgOC43NzcsMTQuMzk4IEM4Ljc3NywxNS41MTMgOS4yMSwxNi43MDkgOS43NDksMTcuMzU5IEM5Ljg1NiwxNy40ODggOS44NzIsMTcuNiA5Ljg0LDE3LjczMSBDOS43NDEsMTguMTQxIDkuNTIsMTkuMDIzIDkuNDc3LDE5LjIwMyBDOS40MiwxOS40NCA5LjI4OCwxOS40OTEgOS4wNCwxOS4zNzYgQzcuNDA4LDE4LjYyMiA2LjM4NywxNi4yNTIgNi4zODcsMTQuMzQ5IEM2LjM4NywxMC4yNTYgOS4zODMsNi40OTcgMTUuMDIyLDYuNDk3IEMxOS41NTUsNi40OTcgMjMuMDc4LDkuNzA1IDIzLjA3OCwxMy45OTEgQzIzLjA3OCwxOC40NjMgMjAuMjM5LDIyLjA2MiAxNi4yOTcsMjIuMDYyIEMxNC45NzMsMjIuMDYyIDEzLjcyOCwyMS4zNzkgMTMuMzAyLDIwLjU3MiBDMTMuMzAyLDIwLjU3MiAxMi42NDcsMjMuMDUgMTIuNDg4LDIzLjY1NyBDMTIuMTkzLDI0Ljc4NCAxMS4zOTYsMjYuMTk2IDEwLjg2MywyNy4wNTggQzEyLjA4NiwyNy40MzQgMTMuMzg2LDI3LjYzNyAxNC43MzMsMjcuNjM3IEMyMS45NSwyNy42MzcgMjcuODAxLDIxLjgyOCAyNy44MDEsMTQuNjYyIEMyNy44MDEsNy40OTUgMjEuOTUsMS42ODYgMTQuNzMzLDEuNjg2IiBmaWxsPSIjYmQwODFjIj48L3BhdGg+PC9nPjwvc3ZnPg==&quot;) 3px 50% / 14px 14px no-repeat rgb(189, 8, 28); position: absolute; opacity: 1; z-index: 8675309; display: none; cursor: pointer; border: none; -webkit-font-smoothing: antialiased;">Save</span><span style="width: 24px; height: 24px; background: url(&quot;data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiA/Pjxzdmcgd2lkdGg9IjI0cHgiIGhlaWdodD0iMjRweCIgdmlld0JveD0iMCAwIDI0IDI0IiB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiPjxkZWZzPjxtYXNrIGlkPSJtIj48cmVjdCBmaWxsPSIjZmZmIiB4PSIwIiB5PSIwIiB3aWR0aD0iMjQiIGhlaWdodD0iMjQiIHJ4PSI2IiByeT0iNiIvPjxyZWN0IGZpbGw9IiMwMDAiIHg9IjUiIHk9IjUiIHdpZHRoPSIxNCIgaGVpZ2h0PSIxNCIgcng9IjEiIHJ5PSIxIi8+PHJlY3QgZmlsbD0iIzAwMCIgeD0iMTAiIHk9IjAiIHdpZHRoPSI0IiBoZWlnaHQ9IjI0Ii8+PHJlY3QgZmlsbD0iIzAwMCIgeD0iMCIgeT0iMTAiIHdpZHRoPSIyNCIgaGVpZ2h0PSI0Ii8+PC9tYXNrPjwvZGVmcz48cmVjdCB4PSIwIiB5PSIwIiB3aWR0aD0iMjQiIGhlaWdodD0iMjQiIGZpbGw9IiNmZmYiIG1hc2s9InVybCgjbSkiLz48L3N2Zz4=&quot;) 50% 50% / 14px 14px no-repeat rgba(0, 0, 0, 0.4); position: absolute; opacity: 1; z-index: 8675309; display: none; cursor: pointer; border: none; border-radius: 12px;"></span></body></html>